app.py

```diff
@@ -7,15 +7,23 @@
 from flask import Flask, session
 
 app = Flask(__name__)
-app.config.update(SECRET_KEY=os.environ.get("FLASK_SECRET_KEY", "some strong secret key"))
-cache = redis.Redis(host="redis", port=6379)
+app.config.update(SECRET_KEY=os.getenv("FLASK_SECRET_KEY", "some strong secret key"))
+cache = redis.Redis(host=os.getenv("REDIS_HOST", "redis"), port=os.getenv("REDIS_PORT", 6379))
```

Comment:
Minor: 'os.getenv' returns string or None. But I think that integer is expected as redis port